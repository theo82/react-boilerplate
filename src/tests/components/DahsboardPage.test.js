import React from 'react';
import { shallow } from 'enzyme';
import DahsboardPage from '../../components/DashboardPage';

test('should render ExpenseDahsboardPage', () => {
    const wrapper = shallow(<DahsboardPage />);
    expect(wrapper).toMatchSnapshot();
});