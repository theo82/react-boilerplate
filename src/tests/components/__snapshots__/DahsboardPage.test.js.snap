// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDahsboardPage 1`] = `
<div>
  Dahsboard page content
</div>
`;
